res/controllers/Denon-Prime-4-scripts.js

```diff
@@ -1,4 +1,5 @@
 var Prime4 = {};
+function Prime4() {}
```

Comment:
These two lines are equivalent. A function is pretty much just an object.
```suggestion
var Prime4 = {};
```